Update to 0.9.15 and disable IPv6 support to fix issues with the built-in HTTP server.

diff --git a/package/libvncserver/libvncserver.hash b/package/libvncserver/libvncserver.hash
index 225c42e..da1c002 100644
--- a/package/libvncserver/libvncserver.hash
+++ b/package/libvncserver/libvncserver.hash
@@ -1,3 +1,3 @@
 # Locally computed:
-sha256  83104e4f7e28b02f8bf6b010d69b626fae591f887e949816305daebae527c9a5  LibVNCServer-0.9.14.tar.gz
+sha256  62352c7795e231dfce044beb96156065a05a05c974e5de9e023d688d8ff675d7  LibVNCServer-0.9.15.tar.gz
 sha256  4d23c8c814e5baf007d854f01d8502e77dc56a41144934e003fb32c4e052d20f  COPYING
diff --git a/package/libvncserver/libvncserver.mk b/package/libvncserver/libvncserver.mk
index 082f838..1db6e7a 100644
--- a/package/libvncserver/libvncserver.mk
+++ b/package/libvncserver/libvncserver.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################

-LIBVNCSERVER_VERSION = 0.9.14
+LIBVNCSERVER_VERSION = 0.9.15
 LIBVNCSERVER_SOURCE = LibVNCServer-$(LIBVNCSERVER_VERSION).tar.gz
 LIBVNCSERVER_SITE = https://github.com/LibVNC/libvncserver/archive
 LIBVNCSERVER_LICENSE = GPL-2.0+
@@ -12,7 +12,7 @@ LIBVNCSERVER_LICENSE_FILES = COPYING
 LIBVNCSERVER_CPE_ID_VENDOR = libvncserver_project
 LIBVNCSERVER_INSTALL_STAGING = YES
 LIBVNCSERVER_DEPENDENCIES = host-pkgconf lzo
-LIBVNCSERVER_CONF_OPTS = -DWITH_LZO=ON
+LIBVNCSERVER_CONF_OPTS = -DWITH_LZO=ON -DWITH_IPv6=OFF

 # only used for examples
 LIBVNCSERVER_CONF_OPTS += \
